docs/.vuepress/config.js

```diff
@@ -12,105 +12,39 @@ module.exports = {
         children: [
-          "/getting-started/voyager",
-          "/getting-started/installation",
-          "/getting-started/join-testnet",
-          "/getting-started/networks"
-        ]
-      },
-      {
-        title: "Cosmos SDK",
-        collapsable: false,
-        children: [
-          ["/sdk/overview", "Overview"],
-          ["/sdk/core/intro", "Core"],
-          "/sdk/core/app1",
-          "/sdk/core/app2",
-          "/sdk/core/app3",
-          "/sdk/core/app4",
-          "/sdk/core/app5",
-          // "/sdk/modules",
-          "/sdk/clients"
-        ]
-      },
-      // {
-      //   title: "Specifications",
-      //   collapsable: false,
-      //   children: [
-      //     ["/specs/overview", "Overview"],
-      //     "/specs/governance",
-      //     "/specs/ibc",
-      //     "/specs/staking",
-      //     "/specs/icts",
-      //   ]
-      // },
-      {
-        title: "SDK by Examples - Simple Governance",
-        collapsable: false,
-        children: [
-          ["/sdk/sdk-by-examples/simple-governance/intro", "Intro"],
-          "/sdk/sdk-by-examples/simple-governance/setup-and-design",
-          "/sdk/sdk-by-examples/simple-governance/app-init",
-          "/sdk/sdk-by-examples/simple-governance/simple-gov-module",
-          "/sdk/sdk-by-examples/simple-governance/bridging-it-all",
-          "/sdk/sdk-by-examples/simple-governance/running-the-application"
-        ]
-      },
-      {
-        title: "Light Client",
-        collapsable: false,
-        children: [
-	  "/lite/",
-	  "/lite/getting_started"
-	]
-      },
-      {
-        title: "Lotion JS",
-        collapsable: false,
-        children: [
-	  ["/lotion/overview", "Overview"]
-	]
-      },
-      {
-        title: "Validators",
-        collapsable: false,
-        children: [
-          ["/validators/overview", "Overview"],
-          ["/validators/security", "Security"],
-          ["/validators/validator-setup", "Validator Setup"],
-          "/validators/validator-faq"
+          "/gaia/installation",
+          "/gaia/join-testnet",
+          "/gaia/networks",
+          "/gaia/validators/overview",
+          "/gaia/validators/security",
+          "/gaia/validators/validator-faq",
+          "/gaia/validators/validator-setup",
+          "/gaia/ledger"
         ]
       },
       {
         title: "Clients",
         collapsable: false,
         children: [
-          ["/clients/service-providers", "Service Providers"]
-        ]
-      },
-      {
-        title: "Resources",
-        collapsable: false,
-        children: [
-          // ["/resources/faq" "General"],
-          "/resources/delegator-faq",
-          ["/resources/whitepaper", "Whitepaper - English"],
-          ["/resources/whitepaper-ko", "Whitepaper - 한국어"],
-          ["/resources/whitepaper-zh-CN", "Whitepaper - 中文"],
-          ["/resources/whitepaper-pt", "Whitepaper - Português"]
-        ]
+      	  "/clients/README",
```

Comment:
can drop `README` otherwise it doesn't render (mystery why this is the case) the readme